FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.DataWriter/Azure/AzureBlobContainerClientFactory.cs

```diff
@@ -30,10 +30,10 @@ public class AzureBlobContainerClientFactory : IAzureBlobContainerClientFactory
 
             _credentialProvider = credentialProvider;
             _loggerFactory = loggerFactory;
-            _storageConfiguration = storageConfiguration.Value;
+            _externalConnectionString = storageConfiguration.Value.ExternalStorageConnectionString;
         }
 
-        public IAzureBlobContainerClient Create(string storeUrl, string containerName, TokenCredentialTypes type = TokenCredentialTypes.External)
+        public IAzureBlobContainerClient Create(string storeUrl, string containerName)
```

Comment:
Seems **storgaeUrl** will not be used when **connectionString** is provided.

Reply:
ConnectionString has higher priority.